omero/developers/cli/extending.rst

```diff
@@ -3,8 +3,17 @@ Extensions
 
 Plugins can be written and put in the ``lib/python/omero/plugins``
 directory. On execution, all plugins in that directory are registered
-with the |CLI|. Alternatively, the :option:`--path` argument can be used to
-point to other plugin files or directories.
+with the |CLI|. 
+
+Alternatively, plugins can be added to any directory ending with
+``omero/plugins``. If this directory is part of the ``PYTHON`` path the |CLI|
+will automatically include them.
+
+For testing purposes the ``--path`` argument can be used to point to other plugin
+files or directories, too.
+
+|CLI| plugins are also pip-installable. Search for ``omero-cli-*`` on `PyPI
```

Comment:
you should use 
```
:pypi:`PyPI <>`

```

Reply:
Since we have mentioned of the plugins on PyPI
I have started looking at the plugin doc so it is similar to what we have for the web apps when pushed on PyPI